spec/features/items_spec.rb

```diff
@@ -101,9 +101,9 @@
 
     it 'is check item edit page' do
       visit edit_item_path(@user_item)
-      expect(page).to have_content('Editing Item')
-      expect(page).to have_link('Back')
-      expect(page).to have_link('Show')
+      expect(page).to have_content('Детали объявления')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.